src/Stack/Ghci.hs

```diff
@@ -186,8 +189,12 @@ ghci opts@GhciOpts{..} = do
               figureOutMainFile bopts mainIsTargets localTargets pkgs0
     -- Build required dependencies and setup local packages.
     stackYaml <- view stackYamlL
+
+    mproject <- view $ configL.to configMaybeProject
+
     buildDepsAndInitialSteps opts (map (T.pack . packageNameString . fst) localTargets)
-    targetWarnings stackYaml localTargets nonLocalTargets mfileTargets
+
```

Comment:
Please don't insert newlines here unless it's logically splitting things into sections (also above)

Reply:
Removed.